src/project-manager.ts

```diff
@@ -359,7 +354,7 @@ export class ProjectManager implements Disposable {
 				);
 			})
 			// Use same scheme, slashes, host for referenced URI as input file
-			.map(filePath => url.format({ ...parts, pathname: filePath.split(/[\\\/]/).map(encodeURIComponent).join('/'), search: undefined, hash: undefined }))
+			.map(filePath => util.path2uri('', filePath))
```

Comment:
But to me, it is always unclear what these functions do exactly. For example, does `uri2path` always return a Unix path? If no, does the path style depend on the host or client OS? Do we have use cases for both? How to differentiate? That's why until now I mostly did the conversion "manually"